Source/santad/SNTDaemonControlController.mm

```diff
@@ -57,10 +58,16 @@
 double watchdogCPUPeak = 0;
 double watchdogRAMPeak = 0;
 
+// Semi-arbitrary limits for temporary Monitor Mode.
+// Min: 1 minute. Max: 30 days.
+static constexpr uint32_t kMinTempMonitorModeMinutes = 1;
+static constexpr uint32_t kMaxTempMonitorModeMinutes = 1 * 60 * 24 * 30;
+
```

Comment:
These already exist in SNTModeTransition.mm, don't they?

Reply:
Good catch. Will consolidate in the next PR.